### PR TITLE
BAH 1381 | Created GHA to generate and publish docker image

### PR DESCRIPTION
**Pull request by umair-fayaz**
1) GHA to build and publish the Bahmni-web docker image to docker hub.
2) Listen to trigger from default config

### CODE REVIEW

**Comment by umair-fayaz** [comment]
`bahmniapps Deploy / Build & Publish Docker Image` failing because of its dependency on [bahmni-package (BAH-1381)](https://github.com/Bahmni/bahmni-package/tree/BAH-1381). For which [PR](https://github.com/Bahmni/bahmni-package/pull/91) has been created

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          path: bahmni-package
+      - name: Download BahmniApps
+        uses: actions/download-artifact@v2
+        with:
+          name: BahmniApps
+          path: bahmni-package/bahmni-web/resources
+      - name: Download default_config.zip
+        run: |
+          curl -H "Accept: application/vnd.github.v3+json" \
+          https://api.github.com/repos/Bahmni/default-config/actions/artifacts | \
```
[the remark this answers is not in the document]
Comment (gsluthra) [reply .github/workflows/ci.yml:62]: thanks. got it. Will a commit to default_config repo then trigger a build of this repository?

[the remark this answers is not in the document]
Comment (umair-fayaz) [reply .github/workflows/ci.yml:62]: Yes @gsluthra. The [on: repository_dispatch](https://github.com/Bahmni/openmrs-module-bahmniapps/blob/BAH-1381/.github/workflows/ci.yml#L8) receives the event from [default_config](https://github.com/Bahmni/default-config/blob/BAH-1380/.github/workflows/ci.yml#L18)

**Comment by abinaya-u** [comment]
Moved docker related resources to individual repos

**Pull request closed without merging**